jina/types/document/uid.py

```diff
@@ -1,13 +1,9 @@
 """
-
 Remarks on the ``id``, we have three views for it
-
 - ``id``: ``str`` is a hex string, for non-binary environment such as HTTP, CLI, HTML and also human-readable. it will be used as the major view.
 - ``bytes``: ``bytes`` is the binary format of str, it has 8 bytes fixed length, so it can be used in the dense file storage, e.g. BinaryPbIndexer, as it requires the key has to be fixed length.
 - ``int``:``int64`` (formerly names ``hash``) is the integer form of bytes, as 8 bytes map to int64 . This is useful when sometimes you want to use key along with other numeric values together in one ndarray, such as ranker and Numpyindexer
```

Comment:
This docstring must be adjusted to state.

Reply:
I think I already did that. Please have a look at the note in docstring. Or do you mean something else?

Reply:
`int` now is not an `int64` anymore, but a regular python `int` which an be of arbitrary length. Also all the "fixed length" documentation does not hold true anymore, right?

Reply:
Right, sorry I missed that.